Prox/Prox/RemoteConfigKeys.swift

```diff
@@ -87,16 +87,34 @@ class RemoteConfigProperty<T> {
 
     var value: T {
         let remoteConfig = FIRRemoteConfig.remoteConfig()
+
+        // rcv is never nil even if there is no key.
+        // rcv.numberValue? never fails either (in light of an invalid number, it defaults to zero).
+        // So, we get the string value, and try parsing it ourselves.
         let rcv = remoteConfig[key]
-        if let value = convert(rcv) {
-            return value
+        if let string = rcv.stringValue {
+            if let value = convert(string) {
+                return value
```

Comment:
Since I assume the empty string is the default value, this will always pass for Strings so we'll never get the type-checked default values of `RemoteConfigPropertyString` – you should remove the default property for Strings and demanded they have plist values.